CORE
Test
--function Test.check --assert-no-exceptions-thrown
line 6 assertion at file Test.java line 6 .*: FAILURE
line 8 assertion at file Test.java line 8 .*: FAILURE
line 10 assertion at file Test.java line 10 .*: FAILURE
line 22 assertion at file Test.java line 22 .*: FAILURE
line 24 assertion at file Test.java line 24 .*: FAILURE
line 26 assertion at file Test.java line 26 .*: SUCCESS
line 28 assertion at file Test.java line 28 .*: SUCCESS
line 30 assertion at file Test.java line 30 .*: SUCCESS
^EXIT=10$
^SIGNAL=0$
--
--
Checks that the `throw` instructions have been replaced by assertions, which
are failing here because they are reachable, and assumptions which prevent
the last three assertions from failing.
